src/client/datascience/ipywidgets/localWidgetScriptSourceProvider.ts

```diff
@@ -106,6 +108,11 @@ export class LocalWidgetScriptSourceProvider implements IWidgetScriptSourceProvi
                 traceError.bind(`Failed to get interpreter details for Kernel/Interpreter ${interpreterOrKernelPath}`)
             );
 
+        if (interpreterInfo && !interpreterInfo.sysPrefix) {
```

Comment:
Work around for Python extension bug https://github.com/microsoft/vscode-python/issues/16355